storage/badger/operation/events.go

```diff
@@ -52,10 +52,29 @@ func RemoveServiceEventsByBlockID(blockID flow.Identifier) func(*badger.Txn) err
 	return removeByPrefix(makePrefix(codeServiceEvent, blockID))
 }
 
+// BatchRemoveServiceEventsByBlockID removes all service events for the given blockID.
+// It returns nil, if all service events are removed (or no service events were found)
+// It returns error, if it runs into any exception
```

Comment:
what errors specifically are expected? What happens if I hand in a block ID that does not exist?

Reply:
Added clarification